src/core/StelLocationMgr.cpp

```diff
@@ -502,6 +505,12 @@ StelLocationMgr::StelLocationMgr()
 	planetName="Earth";
 	planetSurfaceMap=QImage(":/graphicGui/miscWorldMap.jpg");
 	connect(StelApp::getInstance().getCore(), SIGNAL(locationChanged(StelLocation)), this, SLOT(changePlanetMapForLocation(StelLocation)));
+
+	// configure the QGeoPositionInfoSource which can be queried from OS
+	qGeoPositionInfoSource = QGeoPositionInfoSource::createDefaultSource(this);
+	if (qGeoPositionInfoSource)
+		connect(qGeoPositionInfoSource, SIGNAL(positionUpdated(QGeoPositionInfo)),
```

Comment:
Why do you still use this antiquated string-based connection mechanism?

Reply:
Directly taken from Qt6.8 docs.